vertica_python/vertica/column.py

```diff
@@ -149,6 +150,7 @@ def __init__(self, col):
     def convert(self, s):
         if s is None:
             return
+        s = str(s, 'utf-8')
```

Comment:
This needs an explanation, but right now I just have "It seems to work". We might want to do this in a more elegant way.